src/Cache/Adapter.php

```diff
@@ -7,22 +7,31 @@ interface Adapter
     /**
      * @param  string  $key
      * @param  int  $ttl time in seconds
+     * @param  string  $hashKey optional
      * @return mixed
      */
-    public function load(string $key, int $ttl): mixed;
+    public function load(string $key, int $ttl, string $hashKey = ''): mixed;
```

Comment:
Why not just `hash` ?

Reply:
This is the terminology on the Redis interface